draft-ietf-quic-transport.md

```diff
@@ -4486,7 +4535,7 @@ stream before the RST_STREAM frame, and the receiver MUST use the final offset
 to account for all bytes sent on the stream in its connection level flow
 controller.
 
-### Response to a RST_STREAM
+## Response to a RST_STREAM
```

Comment:
These are subsections of "Other Considerations."  If you're going to make them stand-alone sections, I'd move them before that section.  However, I'm more inclined to leave them as-is pending @janaiyengar and @martinthomson's general rearranging of the draft.

Reply:
Some of them were subsection, some were new section, however, i also do not understand why they were subsection. The way it was, was just really confusing and this was the easier fix.